addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/data/TankerkoenigDetailService.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.openhab.binding.tankerkoenig.internal.data;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLConnection;
+
+import org.apache.commons.io.IOUtils;
+import org.openhab.binding.tankerkoenig.internal.config.OpeningTimes;
+import org.openhab.binding.tankerkoenig.internal.config.TankerkoenigDetailResult;
+import org.openhab.binding.tankerkoenig.internal.serializer.CustomTankerkoenigDetailResultDeserializer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+/***
+ * Serivce class requesting detail data from tankerkoenig api and providing result objects
+ *
+ * @author Dennis Dollinger/Jürgen Baginski
+ *
+ */
+public class TankerkoenigDetailService {
```

Comment:
Could that class be merged together with the TankerkoenigService as both are pretty similar and could also share parts of the code?